### PR TITLE
New mapbox functions in PX

### PR DESCRIPTION
**Pull request by nicolaskruchten**
I'll add a couple of tests here and then this should be good to go :)

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
And then we'll need to update some docs to show off the new functions!

File: packages/python/plotly/plotly/express/_chart_types.py [hunk 1]
```diff
@@ -998,6 +998,85 @@ def scatter_mapbox(
...
+    """
+    In a Mapbox choropleth map, each row of `data_frame` is represented by a
+    colored region on a Mapbox map.
+    """
+    return make_figure(
+        args=locals(),
+        constructor=go.Choroplethmapbox,
+        trace_patch=dict(
+            geojson=geojson
+            if not hasattr(geojson, "__geo_interface__")
```
Comment (nicolaskruchten) [inline packages/python/plotly/plotly/express/_chart_types.py]: this is geopandas-specific code so that you can just say `...geojson=geo_df.geometry...` :)

**Comment by emmanuelle** [comment]
Exactly is the doc migration PR is merged then you can add directly the updates to the doc!

**Comment by emmanuelle** [comment]
So the percy script fails because it's missing the `center` argument which has just been introduced here.

**Comment by nicolaskruchten** [comment]
Hah, yes, hurray tests :)

**Comment by emmanuelle** [comment]
:dancer:, thanks!

**Pull request merged**